types/protractor-http-mock/index.d.ts

```diff
@@ -144,31 +146,30 @@ declare namespace mock {
         /**
          * All available request types.
          */
-        type AllRequests<T1, T2> = Get<T1> |
-            PostData<T1, T2> |
-            Post<T1> |
-            Head<T1> |
-            Delete<T1> |
-            Put<T1> |
-            Patch<T1> |
-            Jsonp<T1>;
+        type AllRequests = Get<any> |
+            PostData<any, any> |
+            Post<any> |
+            Head<any> |
+            Delete<any> |
+            Put<any> |
+            Patch<any> |
+            Jsonp<any>;
 
         /**
          * GET request mock.
          */
         interface Get<TResponse> {
             request: {
-                method: Method;
+                method: "GET";
                 path: string;
                 regex?: boolean;
                 params?: Object;
                 queryString?: Object;
                 headers?: Object;
-                interceptedRequest?: boolean;
-                interceptedAnonymousRequest?: boolean;
             };
             response: {
                 status?: number;
+                delay?: number;
```

Comment:
Each response can be delayed now.
https://github.com/atecarlos/protractor-http-mock/blob/77df9c34a6cb837856fdb23e9cfb580a51ea4aec/lib/httpMock.js#L265